File: countAmazon.js
function countArara(n) {
    let odd = 'anane';
    let even = 'adak';
    let countArr = [];
    if (n > 0){
      for (let i = 1; i <=n; i++){
        if (i % 2 == 0){
          countArr.push(even);
        }
        if (i % 2 != 0 && i == n){
          countArr.push(odd);
        }
      }
    }
  return countArr.join(' ');
}

File: count_amazon.py
def count_arara(n):
    odd = 'anane'
    even = 'adak'
    count_arr = []
    if n > 0:
        for i in range(1, n+1):
            if i % 2 == 0:
                count_arr.append(even)
            if i % 2 != 0 and i == n:
                count_arr.append(odd)
    return ' '.join(count_arr)